diff --git a/node_modules/expo-camera/ios/Current/CameraSessionManager.swift b/node_modules/expo-camera/ios/Current/CameraSessionManager.swift
index 1234567..abcdefg 100644
--- a/node_modules/expo-camera/ios/Current/CameraSessionManager.swift
+++ b/node_modules/expo-camera/ios/Current/CameraSessionManager.swift
@@ -132,26 +132,11 @@
     session.beginConfiguration()
     defer { session.commitConfiguration() }
 
-    if delegate.isMuted {
-      for input in session.inputs {
-        if let deviceInput = input as? AVCaptureDeviceInput {
-          if deviceInput.device.hasMediaType(.audio) {
-            session.removeInput(input)
-            return
-          }
-        }
-      }
-    }
-
-    if !delegate.isMuted && delegate.mode == .video {
-      if let audioCapturedevice = AVCaptureDevice.default(for: .audio) {
-        do {
-          let audioDeviceInput = try AVCaptureDeviceInput(device: audioCapturedevice)
-          if session.canAddInput(audioDeviceInput) {
-            session.addInput(audioDeviceInput)
-          }
-        } catch {
-          log.info("\(#function): \(error.localizedDescription)")
+    for input in session.inputs {
+      if let deviceInput = input as? AVCaptureDeviceInput {
+        if deviceInput.device.hasMediaType(.audio) {
+          session.removeInput(input)
+          return
         }
       }
     }
