diff --git a/node_modules/expo-camera/ios/Current/CameraView.swift b/node_modules/expo-camera/ios/Current/CameraView.swift
index 6b4c9eb..a4cc481 100644
--- a/node_modules/expo-camera/ios/Current/CameraView.swift
+++ b/node_modules/expo-camera/ios/Current/CameraView.swift
@@ -630,26 +630,11 @@ public class CameraView: ExpoView, EXAppLifecycleListener,
     session.beginConfiguration()
     defer { session.commitConfiguration() }
 
-    if isMuted {
-      for input in session.inputs {
-        if let deviceInput = input as? AVCaptureDeviceInput {
-          if deviceInput.device.hasMediaType(.audio) {
-            session.removeInput(input)
-            return
-          }
-        }
-      }
-    }
-
-    if !isMuted && mode == .video {
-      if let audioCapturedevice = AVCaptureDevice.default(for: .audio) {
-        do {
-          let audioDeviceInput = try AVCaptureDeviceInput(device: audioCapturedevice)
-          if session.canAddInput(audioDeviceInput) {
-            session.addInput(audioDeviceInput)
-          }
-        } catch {
-          log.info("\(#function): \(error.localizedDescription)")
+    for input in session.inputs {
+      if let deviceInput = input as? AVCaptureDeviceInput {
+        if deviceInput.device.hasMediaType(.audio) {
+          session.removeInput(input)
+          return
         }
       }
     }
